src/main/resources/assets/appliedenergistics2/lang/de_DE.lang

```diff
@@ -277,13 +314,24 @@ gui.tooltips.appliedenergistics2.EmitWhenCrafting=Stoße Redstone-Signal aus, w
 gui.tooltips.appliedenergistics2.ReportInaccessibleItems=Melde unerreichbare Items
 gui.tooltips.appliedenergistics2.ReportInaccessibleItemsYes=Ja: Items, die nicht extrahiert werden können, werden sichtbar.
 gui.tooltips.appliedenergistics2.ReportInaccessibleItemsNo=Nein: Nur extrahierbare Items werden sichtbar.
+gui.tooltips.appliedenergistics2.BlockPlacement=Blockplazierung
+gui.tooltips.appliedenergistics2.BlockPlacementYes=Block wird als Block plaziert.
+gui.tooltips.appliedenergistics2.BlockPlacementNo=Block wird als Item plaziert.
+gui.tooltips.appliedenergistics2.SchedulingMode=Planungsmodus
+gui.tooltips.appliedenergistics2.SchedulingModeDefault=Exportiere das erste Item bis das Netzwerk leer ist, probiere dann das nächste Item.
+gui.tooltips.appliedenergistics2.SchedulingModeRoundRobin=Exportiere im Round-Robin-Modus.
+gui.tooltips.appliedenergistics2.SchedulingModeRandom=Exportiere im Zufallsmodus.
+gui.tooltips.appliedenergistics2.ItemsStored=Items Gespeichert: %s
```

Comment:
I think that's a remnant of the English translation, but yeah, lower case makes sense. Will fix.